EventListener/RateLimitAnnotationListener.php

```diff
@@ -207,4 +208,16 @@ private function getAliasForRequest(FilterControllerEvent $event)
 
         return 'other';
     }
+
+    private function dispatch($eventName, $event)
+    {
+        if (get_class($this->eventDispatcher) === 'Symfony\\Contracts\\EventDispatcher\\EventDispatcherInterface') {
```

Comment:
You should use here instanceof instead of get class. There are multiple types of event dispatchers

Reply:
Ah my bad I somehow had it stuck in my mind that that only works with the ::class constant, which doesn't exist on PHP 5.4, so I went with that instead